app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,31 @@
+@mixin forum-post {
+  .user {
+    float: left;
+    margin-right: 1em;
+    text-align: right;
+    width: 7em;
+  }
+
+  .contents {
+    border: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.